libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2688,7 +2688,28 @@ vector<VariableDeclaration const*> SMTEncoder::stateVariablesIncludingInheritedA
 
 vector<VariableDeclaration const*> SMTEncoder::localVariablesIncludingModifiers(FunctionDefinition const& _function, ContractDefinition const* _contract)
 {
-	return _function.localVariables() + modifiersVariables(_function, _contract);
+	return _function.localVariables() + tryCatchVariables(_function) + modifiersVariables(_function, _contract);
+}
+
+vector<VariableDeclaration const*> SMTEncoder::tryCatchVariables(FunctionDefinition const& _function)
```

Comment:
Parameters from the try-catch clauses are currently not added to the local variables of `FunctionDefinition`s, since they are not `VariableDeclarationStatement`. See `DeclarationRegistrationHelper::endVisit(VariableDeclarationStatement& _variableDeclarationStatement)`.

Maybe this could be changed?

Reply:
Yea we can suggest that